apps/explorer/lib/explorer/chain.ex

```diff
@@ -3568,22 +3738,46 @@ defmodule Explorer.Chain do
     |> Repo.all()
   end
 
+  def address_to_forward_transfers(address_hash, options \\ []) when is_list(options) do
+    necessity_by_association = Keyword.get(options, :necessity_by_association, %{})
+    paging_options = Keyword.get(options, :paging_options, @default_paging_options)
+
+    ForwardTransfer
+    |> where([ft], ft.to_address_hash == ^address_hash)
+    |> order_by([forward_transfer], desc: [forward_transfer.block_number, forward_transfer.index])
+    |> handle_paging_options(paging_options)
+    |> limit(^paging_options.page_size)
+    |> join_associations(necessity_by_association)
+    |> Repo.all()
```

Comment:
I have a couple of questions regarding this query and the next one:
- Are there indexes in the table for the search and order params
- Have you profiled these two queries, in order to verify the query plan is correct and optimal?

Reply:
Hi @otoumas , thanks so much for the review, I will set some time up for us to talk tomorrow. I have some questions.